Dockerfile

```
@@ -41,7 +41,7 @@ RUN wget "https://github.com/mozilla/sops/releases/download/v${SOPS_VERSION}/sop
 
 # Install helmfile
 ARG HELMFILE_VERSION=0.144.0
```

```suggestion
ARG HELMFILE_VERSION=0.145.0
```
New and first release in helmfile/helmfile